serialization.md

```diff
@@ -472,4 +472,40 @@ The resulting Tar archive for `f60c56784b83` has the following entries:
 /etc/.wh.my-app-config
 ```
 
+In addition to expressing that a single entry should be removed from a lower layer, layers may remove all of the children using an opaque whiteout entry.
+An opaque whiteout entry is a file with the name `.wh..wh..opq` indicating that all siblings are hidden in the lower layer.
+Let's take the following base layer as an example:
+
+```
+etc/
+	my-app-config
+bin/
+	my-app-binary
+	my-app-tools
+	tools/
```

Comment:
mind telling (or actually adding a file under `tools/`) that if `tools/` is not-empty we get the opaque only for the topdir `/bin` (this was my misunderstanding also)